modules/infinite_odoscope/js/models/odoscope-article-model.js

```diff
@@ -41,7 +42,8 @@
       }
 
       this.set('loadingIndex', this.get('loadingIndex') + 1);
-      tmpURL = '/lazyloading/node/' + tmpModel.variantID + '/nojs?page=' + this.get('loadingIndex');
+      this.set('currentURL', '/lazyloading/node/' + tmpModel.variantID + '/nojs?page=' + this.get('loadingIndex'));
+      tmpURL = this.get('currentURL');
```

Comment:
Is there a reason for this to be prefixed with tmp? All vars are temporary within their runtime